sdks/java/io/google-cloud-platform/build.gradle

```diff
@@ -244,6 +244,23 @@ task integrationTestKms(type: Test) {
   }
 }
 
+task codeCoverageReport(type: JacocoReport, dependsOn: test) {
```

Comment:
Rather than defining this everywhere we want coverage reports, could we just use a property in applyJavaNature (something kinda like https://github.com/apache/beam/blob/bd86ceed8d1c8813bbd9cdf2a48c429cb7337cb6/buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy#L1610C27-L1610C44)